js/client/modules/@arangodb/replicated-logs.js

```diff
@@ -69,25 +71,25 @@ ArangoReplicatedLog.prototype.status = function() {
   return requestResult.result;
 };
 
-ArangoReplicatedLog.prototype.head = function(limit = 100) {
+ArangoReplicatedLog.prototype.head = function(limit = this.defaultLimit) {
```

Comment:
Instead of doing that you could check for `=== undefined` and then simply omit the query parameter entirely. Then it will use the default value from the methods class and we can remove the duplicate value in line 36.